.env.development

```diff
@@ -1 +1,2 @@
-PORT=3002
\ No newline at end of file
+PORT=3002
+CORS_ORIGIN=http://localhost:3000
```

Comment:
I'd remove in dev, since people can run the local on different ports (I run excal on 5000)

Reply:
i will elave there empty values